code/modules/surgery/other.dm

```diff
@@ -141,6 +141,10 @@
 	user.visible_message("<span class='notice'> [user] has cut away necrotic tissue in [target]'s [affected.name] with \the [tool].</span>", \
 		"<span class='notice'> You have cut away necrotic tissue in [target]'s [affected.name] with \the [tool].</span>")
 	affected.open = 3
+	if(affected.status &= ORGAN_BROKEN)
+		affected.status &= ~ORGAN_BURNT
+		affected.brute_mod /= 1.2
```

Comment:
use the `BURN_WOUND_DAMAGE_MOD` define here. Should also define it in another file than where it is now though

Reply:
Got any suggestions for where to define it? Along with the organ defines?

Reply:
Yeah that looks to be a fine place

Reply:
inb4 brute_mod = 0.99999999999

I can't think of a better way to do this though.